ecl/resource_ecl_sss_approval_request_v1.go

```diff
@@ -0,0 +1,182 @@
+package ecl
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/hashicorp/terraform/helper/validation"
+
+	"github.com/nttcom/eclcloud/ecl/sss/v1/approval_requests"
+)
+
+func resourceSSSApprovalRequestV1() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceSSSApprovalRequestV1Create,
+		Read:   resourceSSSApprovalRequestV1Read,
+		Update: resourceSSSApprovalRequestV1Update,
```

Comment:
Do you have any problem if you remove this field simply? The same applies to Delete.

Reply:
Update can be removed by setting ForceNew to Required fields.
The following error occurs when Delete is removed.
`Delete must be implemented`
So Delete needs to be defined, but setting it to nil makes the method unnecessary.